tests_tf/test_attacks.py

```diff
@@ -40,7 +39,7 @@ def test_generate_np(self):
         perturbation_norm = np.sqrt(np.sum(perturbation**2, axis=1))
         # test perturbation norm
         self.assertTrue(np.allclose(perturbation_norm, self.attack.eps))
-
+        
```

Comment:
looks like the test was not checked in? Also might be easier to manage if different attacks using different files for their test. (could rename the test_attacks.py to be test_vat.py ---sorry didn't catch this earlier in the PR about VAT)